generator/altitude_generator.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/feature_altitude.hpp"
+
+#include "std/string.hpp"
+
+namespace routing
+{
+class IAltitudeGetter
```

Comment:
Ok. Let's rename it.

> Are you ready to rename IAltitudeGetter when it'll have non-virtual methods or fields?

yes.

> Is it interesting for readers that IAltitudeGetter is a pure interface?

Probably sometimes. Readers could consider IAltitudeGetter as an interface.